resources/plugins/arty2/CG_ArtySpotter_V1_0_3.lua

```diff
@@ -0,0 +1,538 @@
+-- Artillery Spotter script
+-- by Carsten Gurk aka Don Rudi
+
+local version = "1.0.3"
+
+-- User configurable variables
+
+local user_fireDelay = 10				-- time to impcat of the rounds
+local user_quantity = 20				-- how many rounds will be fired in a fire for effect task
+local user_spread = 50					-- impact radius of the rounds during fire for effect
+local user_spottingDistance = 15		-- max allowable distance from player to target to prevent cheating. In kilometers.
+
+-- end of user block
+
+-- Script variables
+
+local SINGLE_ROUND = false  			-- pilot called single round on marker (from F10 menu)
+
+local artyCall     = 0 					-- pilot called arty (from F10 menu)
+local artyRadius   = user_spread		-- Artillery Radius
+local adjustRadius = 20  				-- fire adjustment
+local quantity     = 1   				-- Rounds expanded 
+local quantity_effect = user_quantity	-- Rounds expanded during fire for effect
+local tntEquivalent = 15				-- TNT equivalent for explosion
+local fireDelay = user_fireDelay		-- delay til artillery fires in seconds
+
+local firstShotFired = false
```

Comment:
The config isn't applied yet, you need to create a global dictionary in this file that corresponds with the names in arty-config.lua